tests/test_syncs.py

```diff
@@ -0,0 +1,265 @@
+import pytest
+from httpx import Response
+from prefect import flow
+
+from prefect_census.credentials import CensusCredentials
+from prefect_census.runs import (
+    CensusSyncRunCancelled,
+    CensusSyncRunFailed,
+    CensusSyncRunTimeout,
+)
+from prefect_census.syncs import (
+    CensusSyncTriggerFailed,
+    trigger_census_sync,
+    trigger_census_sync_run_and_wait_for_completion,
+)
+
+
+@pytest.fixture
+def census_credentials():
+    return CensusCredentials(api_key="my_api_key")
+
+
+class TestTriggerCensusSyncRun:
+    async def test_trigger_sync(self, respx_mock, census_credentials):
+        respx_mock.post(
+            "https://app.getcensus.com/api/v1/syncs/45/trigger",
+            headers={"Authorization": "Bearer my_api_key"},
+        ).mock(
+            return_value=Response(
+                200, json={"data": {"sync_run_id": 45, "project_id": 12345}}
```

Comment:
Based on the API docs, `project_id` isn't part of this response.
```suggestion
                200, json={"data": {"sync_run_id": 45}}
```